Library/Formula/sdl_ttf.rb

```diff
@@ -18,6 +18,10 @@ class SdlTtf < Formula
   depends_on "sdl"
   depends_on "freetype"
 
+  # Fix broken TTF_RenderGlyph_Shaded()
+  # https://bugzilla.libsdl.org/show_bug.cgi?id=1433
```

Comment:
It says this is fixed in the latest release? If so would be better to update than patch. Thanks!